doc/chapters/design/index.rst

```diff
@@ -0,0 +1,39 @@
+SoftwareContainer Design
+************************
+
+SoftwareContainer has been designed as a building block enabling sandboxing of application contents in a larger system.
+
+The SoftwareContainer container achieves this by providing a structured way to limit what IPC mechanims are made available to each application. By targetting the IPC mechanisms rather than the applications or services directly, the sandboxing mechanism can be deployed in a non-intrusive way, i.e. completely transparent to both applications and services.
```

Comment:
I think the term IPC is pretty generic and most users will probably guess it means inter-process communication, but a compromise where the first occurrence is spelled out might help some readers.